lib/HTML/Nodes/DefinitionListNode.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\RST\HTML\Nodes;
+
+use Doctrine\RST\Nodes\DefinitionListNode as Base;
+use function count;
+use function sprintf;
+
+class DefinitionListNode extends Base
+{
+    public function render() : string
+    {
+        $html = '<dl>';
+
+        $terms = $this->definitionList->getTerms();
+
+        foreach ($terms as $term) {
+            $classifiers = $term->getClassifiers();
+
+            if ($classifiers === []) {
+                $html .= sprintf('<dt>%s</dt>', $term->getTerm());
```

Comment:
Is this already escaped at this point?

Reply:
No, I need to introduce a better way to deal with escaping instead of doing it manually over and over. I will add it manually here for now though. Then in another PR maybe introduce some kind of html builder dependency. This would be useful for other features too like a way to configure classes to add globally for a specific html tag.